Quadriga/src/main/webapp/WEB-INF/tiles/layouts/skeleton_public.jsp

```diff
@@ -48,17 +48,51 @@
             <ul class="nav navbar-nav">
             </ul>
             <ul class="nav navbar-nav navbar-right">
-              <tiles:insertAttribute name="navigation" />  
+              <tiles:insertAttribute name="navigation" />  		
+            
             </ul>
+
             
           </div><!--/.nav-collapse -->
+    
+    
         </div><!--/.container-fluid -->
-      </nav>
 
-      <tiles:insertAttribute name="content" />
 
 
-    </div> <!-- /container -->
+      </nav>
+     
+    <tiles:insertAttribute name="content" />
+<%-- 
+ <tiles:insertAttribute name="content-visual" /> is  added for displaying content which is
+  specific only to publicnetworkvisual.jsp page.  --%>
+  
+
+
+					<div class="row">
+ <div class="8u skel-cell-mainContent">
+							<div class="content content-left">
+								<!-- Content -->
+								<article class="is-page-content">
+									<tiles:insertAttribute name="content-visual" />
+								</article>
+								<!-- /Content -->
+ 							</div>
+ 						</div>
+						<div class="4u">
+							<div class="sidebar">
+								<!-- Sidebar -->
+								<!-- Recent Posts -->
+								<tiles:insertAttribute name="right-navigation" />
+								<!-- /Recent Posts -->
+								<!-- /Sidebar -->
+							</div>
+						</div> 
+					</div> <!-- end of row -->
+ 
+    </div> 
```

Comment:
You don't want to change the layout for all public pages. Most other pages don't have a side navigation and you don't want to use that space for nothing. Just create a new skeleton for public pages with side navigation and use that for the network page instead of the general one.